package.json

```diff
@@ -5,5 +5,30 @@
   "main": "index.js",
   "repository": "https://github.com/pesto-students/batch-10-tripen.git",
   "license": "MIT",
-  "dependencies": {}
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "lint-staged"
+    }
+  },
+  "scripts": {
+    "lint": "./node_modules/.bin/eslint '**/*{.js,.jsx}'"
```

Comment:
using the locally installed eslint rather than forcing devlopers to install eslint globally